content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -188,3 +188,36 @@ To resolve this, we suggest you change your deployment to one with a replica set
 One simple way for this is to use [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) to launch a free instance that has replica set support out of the box.
 
 There's also an option to run the replica set locally with this guide: https://docs.mongodb.com/manual/tutorial/convert-standalone-to-replica-set
+
+## Type mapping between MongoDB and the Prisma schema
+
+> Alternatively, see the [Prisma schema reference](/reference/api-reference/prisma-schema-reference#model-field-scalar-types) for type mappings organized by Prisma type.
+
+The MongoDB connector maps the [scalar types](/concepts/components/prisma-schema/data-model#scalar-fields) from the Prisma [data model](/concepts/components/prisma-schema/data-model) to MongoDB's native column types as follows:
+
+| Prisma    | MongoDB      |
```

Comment:
The Prisma list here is wrong - it can only contain scalar types that Prisma supports, which would be the same terms as on https://www.prisma.io/docs/concepts/database-connectors/postgresql#prisma-migrate

Reply:
Done as part of other edits